commandline/src/main/java/tech/pegasys/eth2signer/commandline/Eth2SignerCommand.java

```diff
@@ -58,6 +61,13 @@
 
   private static final Logger LOG = LogManager.getLogger();
 
+  @SuppressWarnings("UnusedVariable")
+  @CommandLine.Option(
```

Comment:
Could this be done as a mixin perhaps, so we could use the same command for both the initial and base command?